docs/tutorials/image_signing.md

```diff
@@ -0,0 +1,189 @@
+# How to sign and distribute container images using Podman
+
+Signing container images originates from the motivation of trusting only
+dedicated image providers to mitigate man-in-the-middle (MITM) attacks. One way
+to sign images is to utilize a GNU Privacy Guard ([GPG][0]) key. This technique
+is generally compatible with any OCI compliant container registry like
+[Quay.io][1]. It is worth mentioning that the OpenShift integrated container
+registry supports this signing mechanism out of the box.
+
+[0]: https://gnupg.org
+[1]: https://quay.io
+
+From a technical perspective, we can utilize Podman to sign the image before
+pushing it into a remote registry. After that, all systems running Podman have
+to be configured to retrieve the signature key from a remote server, which can
+be any simple web server. This means that every unsigned image will be rejected
+during an image pull operation. But how does this work?
+
+First of all, we have to create a GPG key pair or select an already locally
+available one:
+
+```bash
+> gpg --list-keys sgrunert@suse.com
+pub   rsa2048 2018-11-26 [SC] [expires: 2020-11-25]
+      XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+uid           [ultimate] Sascha Grunert <sgrunert@suse.com>
+sub   rsa2048 2018-11-26 [E] [expires: 2020-11-25]
+```
+
+Now let’s assume that we run a container registry. For example we could simply
+start one on our local machine:
+
+```bash
+> sudo podman run -d -p 5000:5000 registry
```

Comment:
suggest: registry -> docker://docker.io/library/registry:latest
We're having a background convo on the merits of short vs fully named image names.

Reply:
Do you want to start promoting the `docker://` syntax?

I mean, I’m not at all against unambiguous formats, but that merits discussion has not focused on the transport, and I can think of someone who doesn’t like some letters of that transport specifier :)

(I’m fine with both `docker.io/… and docker://docker.io/…`, just to make sure this is discussed.)

Reply:
Sure, I'm fine with dropping the `docker://` from that.  I just tend to always do that and probably ought to retrain myself.

Reply:
Using `docker://` does avoid various ambiguities. OTOH there’s that “Podman is drop-in compatible” story. I really don’t have an opinion either way.

Reply:
Okay, I changed it to `docker.io/registry`